Projects/Book_store/bookStore/bookOutlet/models.py

```diff
@@ -2,16 +2,41 @@
 
 # Create your models here.
 
+class Country(models.Model):
+    name= models.CharField(max_length=50)
+
+
+class Address(models.Model):
+    country = models.CharField(max_length=50)
```

Comment:
@codal-uchoudhary , shouldn't this be a ForeignKey ? You have already created Country model above at line no 5.

Reply:
we can , but here  Address model refers to the address of Author , so i created one-to-one relation between the Author and Address model. 
and , Country model refers to the countries in which a particular book is published so i created many-to-many relation between the Country and Book model.

... for practice purpose let's do this change.